src/server/api/controllers/port.controller.js

```diff
@@ -34,9 +33,25 @@ const createPort = async ({ body }) => {
   if (ports.length !== 0) {
     throw new HttpError("Bad request", "port already exists!", 409);
   } else {
-    return knex("ports").insert({
-      name: name
-    });
+    return knex("ports")
+      .insert({
+        name: name
+      })
+      .then(portID => {
+        if (waypoints.length !== 0) {
+          return knex("waypoints").insert({
+            port_id: portID[0],
+            suggested_route_id: suggested_route_id || null,
+            lon: waypoints[0].lon,
+            lat: waypoints[0].lat
```

Comment:
`lat` and `lon` have now been renamed in the DB.

Reply:
Changes have been incorporated to match DB